src/app/components/fileupload/fileupload.css

```diff
@@ -44,4 +44,8 @@
     .p-fileupload-filename {
         word-break: break-all;
     }
+    
+    .p-button{
```

Comment:
what is the issue?
what is the description?
do you have screenshots before/after?
It's difficult to guess if you only explain with a title...

Thanks!

Reply:
**Issue** : No margins for buttons in Mobile view in file-upload
**Description** :  Should add margins to buttons so that while wrap up they show gaps
**Screenshots** :
Before 
<img width="306" alt="before" src="https://github.com/primefaces/primeng/assets/112557710/c789f895-2c90-4c63-b6c1-cde9ca9e192b">
After
![after](https://github.com/primefaces/primeng/assets/112557710/087cbee6-09e9-4e8d-9bae-40a87ab3902b)
Got it 
P.S:-
And the issue is #15584 
Added the screenshots in the comments, So didn't here previously. 

You're Welcome..

Reply:
> **Issue** : No margins for buttons in Mobile view in file-upload **Description** : Should add margins to buttons so that while wrap up they show gaps **Screenshots** : Before <img alt="before" width="306" src="https://private-user-images.githubusercontent.com/112557710/345720195-c789f895-2c90-4c63-b6c1-cde9ca9e192b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAwNzUzNTMsIm5iZiI6MTcyMDA3NTA1MywicGF0aCI6Ii8xMTI1NTc3MTAvMzQ1NzIwMTk1LWM3ODlmODk1LTJjOTAtNGM2My1iNmMxLWNkZTljYTllMTkyYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwNFQwNjM3MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMzk3NzE0ZmUyODAyYmEwOGFjNDEzMTUzMjU4NmY4Yjg4MTFmMDRiNjU5MDExMjZlOGVmZmRmNGNiMmE5MDNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zDjsDc2cmkPLMcbkeuuwUe6pwaMH6LlBHTvcjrDl-wI"> After ![after](https://private-user-images.githubusercontent.com/112557710/345720407-087cbee6-09e9-4e8d-9bae-40a87ab3902b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAwNzUzNTMsIm5iZiI6MTcyMDA3NTA1MywicGF0aCI6Ii8xMTI1NTc3MTAvMzQ1NzIwNDA3LTA4N2NiZWU2LTA5ZTktNGU4ZC05YmFlLTQwYTg3YWIzOTAyYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwNFQwNjM3MzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOGYxZWVkMGMxMGE5Mzg4ZDE4YzBiNTYyZjg2ZmZlY2EwOWNlYTU2MTM4ZmMzMzZmNTMzMmVmOWE0M2QwMDYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0PnwVj_kpMFs9akaD7uHJ-3WqwZQqVFp0Q_mIJxK_Vs) Got it P.S:- And the issue is #15584 Added the screenshots in the comments, So didn't here previously.
> 
> You're Welcome..

All the informarion should be in the first message that you wrote. You can edit and post the info there.
Also I have checked the issue that you sent #15584 and is talking about TreeSelect component and is not FileUpload.

Please recheck the issue and in your first message add "Fix #NumberOfYourIssue" (without quotes) to link the PR to your issue.

Reply:
fix: #15884

Reply:
I never expect i would some day get a typo-error in numbers..  
A Thanks for the help and My appologies for the inconvenience tho. 😅

Reply:
Well how about this then--->(added the same 0.5rem(given to left) to bottom)
Before:-
![before_0 5rem_desktop](https://github.com/primefaces/primeng/assets/112557710/8414b847-169b-462c-ae2f-6f98823290bf)
![before_0 5rem](https://github.com/primefaces/primeng/assets/112557710/922b2220-28d5-417a-a7a6-f61aab95ee5f)
After:-
![0 5rem_desktop](https://github.com/primefaces/primeng/assets/112557710/8e21a99e-019c-43c5-ab1b-3ed76c2e4719)
![0 5rem](https://github.com/primefaces/primeng/assets/112557710/4ec3c383-2b3c-4ac4-b76b-841a59da0907)